test/Header/HeaderWrapTest.php

```diff
@@ -111,4 +111,25 @@ public function testCanBeEncoded()
         $res = HeaderWrap::canBeEncoded($value);
         $this->assertTrue($res);
     }
+
+    public function testMultilineWithMultibyteSplitAcrossCharacter()
+    {
+        if (! extension_loaded('imap')) {
+            $this->markTestSkipped('works only with imap extension enabled');
+        }
```

Comment:
perhaps phpunit annotation instead?

```
@requires extension imap
```

Reply:
@glensc Thanks for suggestion! PR updated.